doc/kick/invocation.md

```diff
@@ -0,0 +1 @@
+`=kick @SomeUser` - Begins a poll to mute SomeUser. Parameter must be a tag, not plaintext.
```

Comment:
If you copy the docs from other commands as a template, make sure to change the description so that it describes the new command (kick).

Reply:
yup, missed some text when I copy-pasted. I can fix that